apps/transloco-playground/src/app/custom-transpilers/custom.transpiler.ts

```diff
@@ -0,0 +1,13 @@
+import { HashMap, Translation, TranslocoTranspiler, TRANSLOCO_TRANSPILER } from "@ngneat/transloco";
+
+export class CustomTranspiler implements TranslocoTranspiler {
+  transpile(value: any, params: HashMap<any>, translation: Translation, key: any) {
+    console.log({value, params, translation, key})
+    return value
+  }
+}
```

Comment:
```suggestion
export class CustomTranspiler implements TranslocoTranspiler {
  transpile(value: any, params: HashMap<any>, translation: Translation, key: any) {
    console.log('Custom Transpiler called with:', {value, params, translation, key});
    
    return value;
  }
}
```

Reply:
Good suggestion, I have made this change